Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
```

Comment:
This should probably be separate from the kill command since not everyone is running systemd. Probably have a separate code block after the `kill` example like "For systemd, stop etcd using:"?